FSD6-arrays-Engineer/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace EngineerEfficiencyDemo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            EngineersEfficiency engineersEfficiency= new EngineersEfficiency();
```

Comment:
Instead of using the Console.ReadLine for inputs values, pass it from here with a different set of examples mentioned in the jira case.